src/cljs/athens/devcards/style_guide.cljs

```diff
@@ -1,116 +0,0 @@
-(ns athens.devcards.style-guide
-  (:require
-    [athens.db]
-    [athens.style :refer [color THEME-LIGHT THEME-DARK OPACITIES]]
-    [cljsjs.react]
-    [cljsjs.react.dom]
-    [devcards.core :refer-macros [defcard-rg]]
-    [stylefy.core :as stylefy :refer [use-style]]))
```

Comment:
@shanberg do we have analogs for this in storybook yet? Should make sure we do before deleting this file and have to go looking for it in the future.

Reply:
I'll add it to my list of TODOs

Reply:
I would prefer to not leave the files in because:
- the files themselves are not enough to remember to do this
- we'd need to configure out linting/etc to ignore all these broken files (or to leave all the unused libs and deps in)

Instead I propose to leave these as deferred work items in the ADR (see https://github.com/athensresearch/athens/pull/1564/commits/8c668d2c8b6f673adadd1f539d879ccd8b41590b). WDYT?

Reply:
Ok